Projects/DbCLI/SearchDb.py

```diff
@@ -443,10 +446,11 @@ def poolFromCurs(curs, similarityMethod):
         nbrs = nbrLists[(i, nm)]
         nbrTxt = options.outputDelim.join([nm] + ['%s%s%.3f' % (nmDict[id], options.outputDelim,
                                                                 score) for id, score in nbrs])
+
         if outF:
           print(nbrTxt, file=outF)
     else:
-      labels = ['%s%sSimilarity' % (x[1], options.outputDelim) for x in ks]
+      labels = ['%s%ssSimilarity' % (x[1], options.outputDelim) for x in ks]
```

Comment:
Here you added an additional lower case s. Either revert or switch to 
```
labels = ['{}{}Similarity'.format(x[1], options.outputDelim) for x in ks]
```

Reply:
Yes, that additional `s` got in while reverting to the earlier ones. Reverting to `%s%sSimilarity`